implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -410,7 +410,9 @@ impl IdentitiesState {
     }
 
     pub async fn authenticated_storage(&self) -> Result<LmdbStorage> {
-        Ok(LmdbStorage::new(&self.dir.join("authenticated_storage.lmdb")).await?)
+        let storage_dir = self.dir.join("data");
+        std::fs::create_dir_all(&storage_dir)?;
```

Comment:
Let's move this to the `IdentitiesState` constructor, so this is run just once.

Reply:
Updated.